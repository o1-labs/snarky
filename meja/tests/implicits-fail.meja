type showable('a) = {show : 'a -> string};

let show = fun {{show}} => { show; };

let (i, j) = (show (18), show (true));
