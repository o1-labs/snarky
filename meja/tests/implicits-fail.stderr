File "tests/implicits-fail.meja", line 5, characters 14-36:
Error: Could not find an instance for an implicit variable of type int
                                                                    showable.
